source/fundamentals/auth.txt

```diff
@@ -273,17 +272,26 @@ they are listed:
    .. tab:: Web Identity Token File
       :tabid: web-identity-token-file
 
-      You can use the OpenID Connect (OIDC) token obtained from a web identity provider to authenticate
-      to Amazon Elastic Kubernetes Service (EKS) or other services.
-      To use an OIDC token, create a file that contains your token, then 
-      set the absolute path to this file in an environment variable by using
-      a shell as shown in the following example:
+      You can use the OpenID Connect (OIDC) token obtained from a web
+      identity provider to authenticate to Amazon Elastic Kubernetes
+      Service (EKS) or other services. To use an OIDC token, create or
+      locate the file that contains your token. Then, set the following
+      environment variables:
+
+      - ``AWS_WEB_IDENTITY_TOKEN_FILE``: Set to the absolute path of the
+        file that contains your OIDC token.
+
+      - ``AWS_ROLE_ARN``: Set to the IAM role used to connect to your
+        cluster, for example ``arn:aws:iam::111122223333:role/my-role``.
+     
+      The following shell command sets these environment variables:      
 
       .. code-block:: bash
 
-         export AWS_WEB_IDENTITY_TOKEN_FILE=<absolute path to file containing your OIDC token>
+         export AWS_WEB_IDENTITY_TOKEN_FILE=<path to file that contains OIDC token>
```

Comment:
S: my sense is that 'absolute' is important here, to distinguish this from the relative path, but maybe i'm overthinking

Reply:
A relative path should work fine here, although an absolute path is sometimes recommended.